packages/devextreme/js/ui/date_box/ui.date_box.mask.js

```diff
@@ -331,6 +336,18 @@ const DateBoxMask = DateBoxBase.inherit({
         }
     },
 
+    _handleAmPmSearch(char) {
+        const startString = this._searchValue + char.toLowerCase();
+
+        this._searchValue = startString;
```

Comment:
please explain why did you add this code? it is not clear for me. if i remove it all tests pass